Munna-Khandakar/HFAandPerksbdLandingPage
# Contact page

## [Feature] Add Email-Sending Backend

### Summary
This PR adds backend functionality to send emails from the Contact form in `ContactSection` using Nodemailer with Gmail.

### Changes
- Added `/api/send-email` endpoint for email handling.
- Implemented form submission handling with success/error feedback.
- Added validation for required fields.

### Testing
1. Fill out the Contact form and submit.
2. Verify email is sent and success/error messages display correctly.

### Notes
- Requires `nodemailer` and `dotenv`.
- Ensure Gmail credentials are set in environment variables.
